grommet/grommet
# Remove findDomNode from infinite scroll

#### What does this PR do?
Removes findDomNode from infiniteScroll.js that causes an error in React.Strictmode. Changes Ref component to use react.forwardref.

#### Where should the reviewer start?
InfiniteScroll.js

#### What testing has been done on this PR?
Tested with infinite scroll examples in storybook and using this
```
    <React.StrictMode>
      <Box height="small" overflow="auto">
        <Select
          options={["Device 1", "Device 2", "Device 3"]}
          value={value}
          onChange={({ option }) => setValue(option)}
        />
      </Box>
    </React.StrictMode>
```
#### How should this be manually tested?
I would test to check if inifinite scroll behaves as it should, and with the code pasted above.

#### Any background context you want to provide?
I'm a little unsure if this is the correct solution. I compared my local storybook to the current on up, and everything behaved the same way. I did some research, and the accepted replacement for findDomNode was to use forwardRef.

#### What are the relevant issues?
#3993 

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
No

#### Is this change backwards compatible or is it a breaking change?
Should be backwards compatible.
